std_msgs/Header header
iiwas_control/BsplineSegmentMsg[] segments